### PR TITLE
[Doc][Connector-v2][KafkaSink]Supplementary documents

### PR DESCRIPTION
**Pull request by lightzhao**
## Purpose of this pull request
Supplementary documents with kafkaSink

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/sink/Kafka.md [hunk 1]
```diff
@@ -84,6 +84,10 @@ If name is set as the key, then the hash value of the name column will determine
 
 If not set partition key fields, the null message key will be sent to.
 
+The format of the message key is json, If name is set as the key, for example '{"name":"Jack"}'.
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/Kafka.md:87]: Hi, I don't get the mean about this part.

Comment (lightzhao) [reply docs/en/connector-v2/sink/Kafka.md:87]: This means that after setting the "partition_key_fields" attribute, the format of the kafka message key is json, and an example is listed.

Comment (Hisoka-X) [reply docs/en/connector-v2/sink/Kafka.md:87]: Maybe you should change to `If the format of the message is json, and the name field be setted as the partition_key_fields, for example '{"name":"Jack"}'. Then 'Jack' will be used for partition`.

Comment (Hisoka-X) [reply docs/en/connector-v2/sink/Kafka.md:87]: Maybe you should change to `If the format of the message is json, and the name field be setted as the partition_key_fields, for example '{"name":"Jack"}'. Then `Jack` will be used for partition`.

Comment (lightzhao) [reply docs/en/connector-v2/sink/Kafka.md:87]: This is not the format of the message. This is the key format of the message written to kafka after setting the "partition_key_fields" attribute. For example, if the "partition_key_fields" attribute is set to name, the key of the message kafka message should be '{"name": "Jack"}'.

Comment (Hisoka-X) [reply docs/en/connector-v2/sink/Kafka.md:87]: I checked the code, you are right.

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by TyrantLucifer — APPROVED** [review]

**Pull request merged**